test/shopify-cli/theme/syncer_test.rb

```diff
@@ -757,6 +758,47 @@ def test_broken_file_when_file_is_not_broken
         refute(@syncer.broken_file?(file))
       end
 
+      # Case where file is an asset but has errors, resp is: { errors => { asset => ["..."]}}
+      def test_parse_valid_theme_asset_with_errors_bulk
+        op = stub(file: "sections/footer.liquid", method: "update")
+        expected_result = ["Something is wrong with this theme file."]
+        body = {
+          "errors" => {
+            "asset" => expected_result,
+          },
+        }
+        err = ShopifyCLI::Theme::ThemeAdminAPIThrottler::AssetUploadError.new(body, response: { body: body })
+
+        result = @syncer.send(:parse_api_errors, op, err)
+        assert_equal(result, expected_result)
+      end
+
+      # Case where file is not a valid theme asset, resp is: { errors => ["..."]}
+      def test_parse_invalid_theme_asset_with_errors_bulk
```

Comment:
Last 2 tests are for when we have an invalid theme asset. Error comes in form:
{errors => [..] || "..."}